packages/mask/src/plugins/Trader/apis/trending/index.ts

```diff
@@ -85,7 +85,10 @@ export async function getNonFungibleTokenActivities(
     chainId: Web3Helper.ChainIdAll,
     contractAddress: string,
     cursor: string,
-): Promise<{ content: NonFungibleTokenActivity[]; cursor: string } | undefined> {
+): Promise<
+    | { content: Array<NonFungibleTokenActivity<Web3Helper.ChainIdAll, Web3Helper.SchemaTypeAll>>; cursor: string }
```

Comment:
Use Web3Helper.NonFungiblrTokenActivity.

Reply:
`NonFungibleTokenActivity ` is defined at `packages/web3-shared/base/src/specs/index.ts`.

Reply:
Done